common-utils/src/main/java/eu/arrowhead/common/http/HttpService.java

```diff
@@ -141,7 +141,7 @@ public <T, P> T sendRequest(final UriComponents uri, final HttpMethod method, fi
 		try {
 			final WebClient client = createWebClient(usedClient);
 			final RequestBodySpec spec = client.method(method)
-					.uri(uri.toUri());
+					.uri(uri.toUri()); 
```

Comment:
Is this one just one space addition? I think Checkstyle shows a warning because of this.